### DISCUSSION TITLE
iOS HealthKit usage for Xamarin

### DISCUSSION THREAD

**Discussion by vlajkoe** [DevTrevi/DgcReader#92] [Q&A]
Hi,

since implementing this library for Italian check, the AppStore rejects my iOS App on the basis of using HealthKit key.
I went through the whole project of mine and could not find any usage of HealthKit, so I was wondering if there are any keys related to usage of HealthKit in this library for Xamarin.

Thanks!

**Comment by DevTrevi** [comment]
Hi @vlajkoe, I confirm that the library does not reference any HealthKit functionality, nor any iOS specific api
